lxd/main.go

```diff
@@ -894,6 +897,23 @@ they otherwise would.
 		if !askBool("Would you like stale cached images to be updated automatically? (yes/no) [default=yes]? ", "yes") {
 			imagesAutoUpdate = false
 		}
+
+		if askBool("Would you like to create a new network bridge (yes/no) [default=yes]? ", "yes") {
+			bridgeName = askString("What should the new bridge be called [default=lxdbr0]? ", "lxdbr0", networkValidName)
+			bridgeIPv4 = askString("What IPv4 subnet should be used (CIDR notation, “auto” or “none”) [default=auto]? ", "auto", func(value string) error {
```

Comment:
I didn't because the line would then end up wrapping for most users making the UX a bit crap and because if we give an example people are likely to just use it, getting us back to the kind of problem we had before (subnet conflicts breaking connectivity).